src/content/self-hosted/administration/configuration.mdx

```diff
@@ -47,6 +47,26 @@ subdomain: "example.com"
 
 After installation, we recommend that you create a DNS entry for `*.$SUBDOMAIN`, pointing to the public address of your load balancer.
 
+### publicOverride
+
+Use this property to override where Okteto is available. This option replaces `okteto.$SUBDOMAIN` with you [FQDN](https://en.wikipedia.org/wiki/Fully_qualified_domain_name) of choice.
+
+When set, an additional component is deployed for [Okteto Private Endpoints](cloud/private-endpoints.mdx) feature. You can configure its settings at [`privateEndpoints`] section.
```

Comment:
What settings can be configured? How is this different from when not using publicOverride?